addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -292,17 +310,3 @@ def test_gap_in_sequence_warning(self):
         moves[6].button_draft()
         moves[6].button_cancel()
         self.assertTrue(journal._query_has_sequence_holes())  # gap due to canceled move using a sequence, gap warning
-
-    def test_bank_journal_misc_operations_with_payments(self):
```

Comment:
@Megaaaaaa is there a reason this test was removed?

Reply:
I just tested and the test doesn't break so I don't think so, this might be a mistake from a rebase i did. If there is a reason, i can't remember it.

Reply:
OK I bring it back in #167660